lib/console.js

```diff
@@ -35,20 +37,34 @@ function Console(stdout, stderr) {
   }
 }
 
+function write(ignoreErrors, stream, string) {
+  if (!ignoreErrors) return stream.write(string);
+
+  try {
+    stream.write(string, (err) => {
+      if (err) {
+        stream.once('error', () => {});
+      }
+    });
```

Comment:
this 'error' function is added over and over again at every write.
I think the function should be added after stream.write synchronously, and then removed in the callback.
Beware of not allocating any new callbacks here, we will be adding strain to the gc for nothing (they can all be top level).